safe/gui/tools/minimum_needs/needs_profile.py

```diff
@@ -162,6 +162,15 @@ def sort_by_locale(unsorted_profiles, locale):
         profiles = sort_by_locale(profiles, self.locale)
         return profiles
 
+    def precision_of(self, num_txt):
+        """Get the precision by counting chars to right of '.' .
+
+        :returns: The precision of float parameters.
```

Comment:
:param number_as_text: A textual representation of the number whose precision we wish to determine. 
:type number_as_text: basestring